plugins/woocommerce/src/Admin/API/Reports/DataStore.php

```diff
@@ -1374,6 +1375,10 @@ protected function get_attribute_subqueries( $query_args ) {
 					// Assume these are a custom attribute slug/value pair.
 					$meta_key   = esc_sql( $attribute_term[0] );
 					$meta_value = esc_sql( $attribute_term[1] );
+					$attr_term  = get_term_by( 'slug', $meta_value, $meta_key );
+					if ( false !== $attr_term ) {
+						$term_id = $attr_term->term_id;
+					}
```

Comment:
`$term_id` wasn't previously being used, but now it is, this makes sure it is set in both instances.